streamlit_pills/__init__.py

```diff
@@ -16,10 +16,11 @@ def pills(
     label: str,
     options: Iterable[str],
     icons: Iterable[str] = None,
-    index: Union[int, None] = 0,
+    index: Union[int, Iterable[int], None] = 0,
```

Comment:
I ran the streamlit-pills based on this code, but it seems that there is a potential for an error to occur in the **60th line**, `if index and not isinstance(index, list):`, if no value is passed to the index option. 

So I think it would be good if you could modify it as follows.
```
index: Union[int, Iterable[int], None] = 0
```
modified
```
index: Union[Iterable[int], None] = [0]
```